src/components/HistorySideBar.tsx

```diff
@@ -102,11 +106,32 @@ export const HistorySideBar: React.FunctionComponent<IHistorySideBarProps> = (
       }
     };
 
+  /**
+   * Commit info for 'Uncommitted Changes' history.
+   */
+  const uncommitted = React.useMemo<Git.ISingleCommitInfo>(() => {
+    return {
+      author: props.trans.__('You'),
+      // INDEX or WORKING special ref
+      commit: `${+(props.model.selectedHistoryFile?.status === 'staged')}`,
+      pre_commit: 'HEAD',
+      is_binary: !!props.commits[0]?.is_binary,
```

Comment:
Same here, for clarity let use the nullish coalescing operator `??`

```suggestion
      is_binary: props.commits[0]?.is_binary ?? false,
```